modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java

```diff
@@ -292,6 +292,20 @@ public void saveAccountAndLogin(final String accountData, final String password
         }
     }
 
+    @ReactMethod
+    public void saveAccountAndLoginWithKeycard(final String accountData, final String password , final String config, final String chatKey) {
+        Log.d(TAG, "saveAccountAndLoginWithKeycard");
+        String finalConfig = prepareDirAndUpdateConfig(config);
+        String result = Statusgo.saveAccountAndLoginWithKeycard(accountData, password, finalConfig, chatKey);
```

Comment:
why does it take a chatKey instead of subAccountsData like saveAccountAndLogin?

Reply:
@yenda 
we need to pass whisper private key from keycard to status-go. Also, here is the PR where it was done on status-go side
https://github.com/status-im/status-go/pull/1587